src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -0,0 +1,120 @@
+                     :as-text (fn [data] (data "library:datasetDescription"))
+                     :content-renderer (fn [data]
+                                         [:a {:href (str "#workspaces/"
+                                                         (js/encodeURIComponent (str (data "namespace") ":" (data "name"))))}
+                                          (data "library:datasetName")])}
+                    {:header "Phenotype/indication" :starting-width 200
+                     :sort-by clojure.string/lower-case}
+                    {:header "Data Use Restrictions" :starting-width 200
+                     :sort-by clojure.string/lower-case}
+                    {:header "# of Participants" :starting-width 150}]
+          :pagination (react/call :pagination this)
+          :->row (fn [item]
+                   [ item
+                    (item "library:indication")
+                    (item "library:dataUseRestriction")
+                    (item "library:numSubjects")
+                    ])}]]))
+   :set-filter-text
+   (fn [{:keys [refs]} new-filter-text]
+     (react/call :update-query-params (@refs "table") {:filter-text new-filter-text}))
+   :pagination
+   (fn [{:keys [state]}]
+     (fn [{:keys [current-page rows-per-page filter-text]} callback]
+       (endpoints/call-ajax-orch
+         (let [from (* (- current-page 1) rows-per-page)]
+           {:endpoint
+              endpoints/search-datasets
+            :payload
+            (cond
+              (or (nil? filter-text) (= "" filter-text)) {:from from :size rows-per-page}
+              :else
+              {:searchTerm filter-text :from from :size rows-per-page})
+            :headers utils/content-type=json
+            :on-done
+            (fn [{:keys [success? get-parsed-response status-text]}]
+              (if success?
+                (let [response (get-parsed-response) total (response "total") results (response "results")]
+                  (swap! state assoc :total total)
+                  (callback {:group-count total
+                             :filtered-count total
+                             :rows results}))
+                (callback {:error status-text})))}))))})
+
+
+(react/defc FilterSection
+  {:render
+   (fn [{:keys [props state this]}]
+     [:div {:style {:flex "1 0 auto" :marginRight "1em"}}
+      [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Filters: "]
+      [:div {:style {:padding "4"}}]
+      [:div {:style {:display "inline-flex" :background (:background-light style/colors) :padding "16px 8px 10px 8px"}}
+       (style/create-text-field
+         {:ref "search-field"
+          :style {:borderRadius "3px 0 0 3px" :marginBottom 0}
+          :onKeyDown (common/create-key-handler [:enter] #(react/call :apply-filter this))
+          })
+       [comps/Button {:icon :search
+                      :onClick #(react/call :apply-filter this)
+                      :style {:borderRadius "0 3px 3px 0"}}]
+       ]]
```

Comment:
I don't like that this is completely copy/pasted.  This component should be pulled out into components.cljs.